content/en/apps/guides/data/analytics/building-dbt-models.md

```diff
@@ -0,0 +1,340 @@
+
+In CHT Sync config, set the URL of this repository to the `CHT_PIPELINE_BRANCH_URL` [environment variable]({{< relref "apps/guides/data/analytics/environment-variables" >}}), either in ´.env´ if using ´docker compose´, or in ´values.yaml´ if using Kubernetes.
+
+### Deploying models
+
+CHT Sync automatically checks for updates to the DBT project at `CHT_PIPELINE_BRANCH_URL`
+Use the main branch for changes that should be released; they will applied as soon as they are pushed to the repository.
+For models that are in development, any other branch can be used.
+
+Rebuilding tables can take some time (a rough estimate is several hours for tables between 1M and 10M rows, upt to 24H for tables between 10M and 100M rows) so plan for affected dashboards to be out of date when releasing new changes to models; only changed tables and their dependencies will be rebuilt.
+
+New releases of the base models are rare, but schema changes to the CHT or new features may require changes.
+Generally, these changes will be backwards compatible so that a new release of the CHT, or to the base models, does not break application specific models.
+
+When it is necessary to update the base models, update the version tag in the dependency that refers to cht-pipeline, make any changes to application specific models that are necessary, rerun any unit tests, and push to the main branch to release. 
+
+
+### Testing models and dashboards
+
+It is highly encouraged to write dbt [tests](https://docs.getdbt.com/docs/build/data-tests) for application specific models to ensure that they are accurate and to avoid releasing broken models. Examples can be found in [CHT Pipeline](https://github.com/medic/cht-pipeline/tree/main/test).
+
+
+## Base Models
+
+All tables contain a `uuid` which is the primary key for the table; it is also the `_id` from the source CouchDB document.
+
+{{< figure src="cht-pipeline-er.png" link="cht-pipeline-er.png" class=" center col-16 col-lg-12" >}}
+
+### `couchdb`
+All documents are stored in the `couchdb` table; downstream models move fields from the document into fields and index them.
+This table can be used to get any field from a CouchDB document; however, it is recommended that the columns from downstream models be used instead for performance reasons.
+|Field|Description|
+|--|--|
+|`uuid`|CouchDB's unique identifier of the record|
+|`type`|The general type of the document, see below|
+|`doc`| JSON of the source document|
+
+### `data_record`
+All form responses are stored in the `data_record` table; see more details [in the database schema conventions]({{< ref "core/overview/db-schema#reports" >}}).
+This table contains columns for the contact who made the report, the parent of that contact, the report data, and a copy of fields in `jsonb` format.
+It is not recommended to use the fields column directly, but instead to add one new model for each form, moving the relevant fields for that form into columns
+
+|Field|Description|
```

Comment:
The [columns](https://github.com/medic/cht-pipeline/blob/b4ecea6ac3abddd2d49e66dbd30a87409eae91dd/models/forms/data_record.sql#L21-L22) have changed and should be updated.